nodes/bm_inventory_r4pac08.json

```diff
@@ -1113,6 +1114,26 @@
                         "port_id": "twentyfivegige 1/7/3",
                         "switch_id": "4c:76:25:e7:e3:42"
                     }
+                },
+                {
+                    "address": "f4:52:14:01:b9:e1",
+                    "physical_network": "datacentre",
+                    "pxe_enabled": "False",
+                    "local_link_connection": {
+                        "switch_info": "MOC-R4PAC08-SW-TORS-A",
+                        "port_id": "fortygige 1/17/1",
+                        "switch_id": "d8:9e:f3:ae:e5:a2"
+                    }
+                },
+                {
+                    "address": "f4:52:14:01:b9:e2",
+                    "physical_network": "datacentre",
+                    "pxe_enabled": "False",
+                    "local_link_connection": {
+                        "switch_info": "MOC-R4PAC08-SW-TORS-B",
+                        "port_id": "fortygige 1/17/1",
```

Comment:
the port_id is the same every where.

Reply:
That's okay, because it's port 17 on each switch of each rack (1 node in each rack, 2 switches per)

Reply:
right, my bad. it's on different switches.